CS/ContextMenu_CS.txt

```diff
@@ -146,14 +146,14 @@ ContextMenu_CS = {
     ContextMenu_ExtinguishFire = "Uhasit oheň",
     ContextMenu_EyeLeft = "Nasadit na levé oko",
     ContextMenu_EyeRight = "Nasadit na pravé oko",
-    ContextMenu_FannyPack_WearBack = "Dozadu",
-    ContextMenu_FannyPack_WearFront = "Dopředu",
+    ContextMenu_FannyPack_WearBack = "Nosené vzadu",
+    ContextMenu_FannyPack_WearFront = "Nosené vpředu",
```

Comment:
To mi moc nesedí 🤔
Víš že je to tenhle textový element?
![screenshot_16-01-2022_13-24-40](https://user-images.githubusercontent.com/97183670/149659867-3f331eb0-7a82-4118-87b2-27c8ac875be0.png)

Reply:
![20220116145606_1](https://user-images.githubusercontent.com/29583652/149663172-821d30a5-95c0-4104-bda7-469c67dded74.jpg)
Ale i o tento

Reply:
Tak co
`Nasadit dozadu`
`Nasadit dopředu`
?

Reply:
to dozadu tak tahá za oči O.O